scripts/setup_host.sh

```diff
@@ -319,6 +332,17 @@ function check_os() {
 }
 
 function check_network(){
+    #Check if you are able to access external website without actually downloading it
+    wget --timeout=3 --tries=1 -q --spider https://github.com
+    if [ $? -ne 0 ]; then
+        echo "access https://google.com failed!"
```

Comment:
typo? github.com or google.com?

Reply:
yes, sorry, it should have been github.com. I had previously tried google.com but then realized that github.com is a better website to check the connection as we rely on github.com to pull few other projects.